app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ConversationsController < ApplicationController
+  before_filter :authenticate_user!, :mailbox
+  before_filter :conversation, except: [:index, :new, :create]
+
+  def create
+    recipient_ids = conversation_params[:recipients].split(',')
```

Comment:
You may be able to skip the call to `.split`, see https://github.com/xudongcamsys/homefoodie/pull/6/files#r32397782